src/Modes/Insert.ts

```diff
@@ -41,6 +41,16 @@ export class ModeInsert extends Mode {
         });
     }
 
+    enter() {
+      super.enter();
+      this.startRecord();
+    }
+
+    exit() {
```

Comment:
Can we define the returning type `void` so someone won't accidentally return something and use it.